docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,210 @@
+
+1. Download and install the
+[nRF Command Line Tools](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools).
+2. Clone OpenThread repository into the current directory:
+
+        $ git clone https://github.com/openthread/openthread.git
+        
+3. Install OpenThread dependencies:
+
+        $ cd openthread
+        $ ./script/bootstrap
+4. Set up build environment:
+
+        $ ./bootstrap
+5. Build OpenThread for the nRF52840 DK:
+
+        $ make -f examples/Makefile-nrf52840
+
+6. Convert the resulting RCP image to HEX format:
+
+        $ arm-none-eabi-objcopy -O ihex output/nrf52840/bin/ot-rcp output/nrf52840/bin/ot-rcp.hex
+7. Flash the RCP firmware:
+
+        $ nrfjprog --chiperase --program output/nrf52840/bin/ot-rcp.hex --reset
+8. Disable Mass Storage feature on the device so that it doesn't [interfere](https://github.com/openthread/openthread/blob/master/examples/platforms/nrf528xx/nrf52840/README.md#mass-storage-device-known-issue) with core RCP functionalities. Note that power-cycle of the device is required to apply the changes. The setting will remain valid even if you flash another firmware onto the device.
+
+        $ JLinkExe
+        J-Link>MSDDisable
+        Probe configured successfully.
+        J-Link>exit
+
+
+</details>
+
+## Configuring PC as a Thread border router
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+To make your PC work as a Thread border router, you need to complete the following tasks which can be further broken down into smaller steps:
+
+1. Form a Thread network using OpenThread Border Router Docker image and configure IPv6 packet routing to the network.
+2. Form a Wi-Fi network.
+
+To form a Thread network, perform the following steps:
+
+1. Create an IPv6 network for an OTBR Docker container:
+
+        $ docker network create --ipv6 --subnet 2001:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
+2. Start an OTBR Docker container using the command below with _rcp-dk_ in the last line replaced with the device node name of the nRF52840 DK running the RCP firmware, e.g. _/dev/ttyACM0_:
+
+        $ docker run -it --rm --privileged --network otbr -p 8080:80 \
+                --sysctl "net.ipv6.conf.all.disable_ipv6=0 net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1" \
+                --volume rcp-dk:/dev/ttyACM0 openthread/otbr --radio-url spinel+hdlc+uart:///dev/ttyACM0
+3. Open `http://localhost:8080/` address in a web browser.
+4. Click __Form__ button on the left menu.
+5. Notice that On-Mesh Prefix is `fd11:22::`. It will be used later to configure IPv6 packet routing.
+6. Click __Form__ button at the bottom to form a new Thread network using the default settings.
+7. Check the output from `docker network inspect otbr` to learn which IPv6 address was assigned to the OTBR Docker container. Most likely, it will be `2001:db8:1::2`.
```

Comment:
Output is pretty huge, what about to be more specific: `docker network inspect otbr | grep IPv6Address`